android/src/main/java/com/reactnativereceivesharingintent/ReceiveSharingIntentModule.java

```diff
@@ -26,22 +24,26 @@ public ReceiveSharingIntentModule(ReactApplicationContext reactContext) {
     super(reactContext);
     this.reactContext = reactContext;
     Application applicationContext = (Application) reactContext.getApplicationContext();
-    receiveSharingIntentHelper = new ReceiveSharingIntentHelper(applicationContext);
+    receiveSharingIntentHelper =
+      new ReceiveSharingIntentHelper(applicationContext);
   }
 
-
   protected void onNewIntent(Intent intent) {
     Activity mActivity = getCurrentActivity();
-    if(mActivity == null) { return; }
+    if (mActivity == null) {
+      return;
+    }
     oldIntent = mActivity.getIntent();
     mActivity.setIntent(intent);
   }
 
   @RequiresApi(api = Build.VERSION_CODES.KITKAT)
```

Comment:
Just to add as note, if we decide to drop deprecated API logics, this should be still here because RequiresApi checks for `equal or greater than`